.gitignore

```diff
@@ -10,3 +10,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Dependencies
+vendor/
```

Comment:
You might want to consider using a more comprehensive base such as: https://www.gitignore.io/api/go,vim,macos,visualstudiocode

Reply:
Thanks! I'll update it in next PR.